samples/PSFScriptingSample/readme.md

```diff
@@ -0,0 +1,13 @@
+# Scripting sample
+This solution contains example scripts and configurations to make an application to see how scripting works in the Package Support Framework.
```

Comment:
>This [](start = 0, length = 4)

This solution shows examples of the scripting capabilities in Package Support Framework. It contains configuration and application samples.